tests/src/test/scala/whisk/core/database/test/RemoveLogsTests.scala

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2015-2016 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package whisk.core.database.test
+
+import java.io.File
+import java.time.Instant
+import java.time.temporal.ChronoUnit
+
+import org.junit.runner.RunWith
+import org.scalatest.FlatSpec
+import org.scalatest.junit.JUnitRunner
+
+import common.StreamLogging
+import common.TestUtils
+import common.WhiskProperties
+import common.WskActorSystem
+import whisk.core.entity.ActivationId
+import whisk.core.entity.EntityName
+import whisk.core.entity.EntityPath
+import whisk.core.entity.Subject
+import whisk.core.entity.WhiskActivation
+import whisk.core.entity.ActivationLogs
+
+@RunWith(classOf[JUnitRunner])
+class RemoveLogsTests
+    extends FlatSpec
+    with DatabaseScriptTestUtils
+    with StreamLogging
+    with WskActorSystem {
+
+    val designDocPath = WhiskProperties.getFileRelativeToWhiskHome("ansible/files/logCleanup_design_document_for_activations_db.json").getAbsolutePath
+    val removeLogsToolPath = WhiskProperties.getFileRelativeToWhiskHome("tools/db/deleteLogsFromActivations.py").getAbsolutePath
+
+    /** Runs the clean up script to delete old activations */
+    def removeLogsTool(dbUrl: String, dbName: String, days: Int, docsPerRequest: Int = 20) = {
+        println(s"Running removeLogs tool: $dbUrl, $dbName, $days, $docsPerRequest")
+
+        val cmd = Seq(python, removeLogsToolPath, "--dbUrl", dbUrl, "--dbName", dbName, "--days", days.toString, "--docsPerRequest", docsPerRequest.toString)
+        val rr = TestUtils.runCmd(0, new File("."), cmd: _*)
```

Comment:
Wondering if we want to replace `TestUtils` in new tests, it's the old Java stuff I think?

`cmd ! shouldBe 0` should be the same, using the correct `scala.Process` related imports. Won't sit on that change though, but might be worth a try 😁